packages/@react-spectrum/menu/docs/MenuTrigger.mdx

```diff
@@ -71,27 +71,39 @@ import CropRotate from '@spectrum-icons/workflow/CropRotate';
 import Slice from '@spectrum-icons/workflow/Slice';
 import {Text} from '@react-spectrum/text';
 
-<MenuTrigger trigger="longPress">
-  <ActionButton
-    aria-label="Crop tool"
-    onPress={() => alert('Cropping!')}>
-    <Crop />
-  </ActionButton>
-  <Menu>
-    <Item>
-      <CropRotate />
-      <Text>Crop Rotate</Text>
-    </Item>
-    <Item textValue="Slice">
-      <Slice />
-      <Text>Slice</Text>
-    </Item>
-    <Item textValue="Clone stamp">
-      <CloneStamp />
-      <Text>Clone Stamp</Text>
-    </Item>
-  </Menu>
-</MenuTrigger>
+function Tools() {
```

Comment:
now that i see it in action, technically we should apply selection like we do to action groups to the action button trigger when clicked and unset it when clicked again, it's a toggle
I don't think we support that on ActionButton's though, so... maybe your original example would be less confusing

don't know what do you think?

Reply:
In actiongroup it looks like we apply a classname for is-selected

Reply:
yeah, thats a good point, I'll revert it to the original example. Really this example is just to show the functionality of longPress so keeping it simple is more beneficial IMO